stix2/base.py

```diff
@@ -124,11 +124,20 @@ def __len__(self):
 
     # Handle attribute access just like key access
     def __getattr__(self, name):
-        return self.get(name)
+        # If the requested attribute is not None return its value
```

Comment:
Would it work to just call `self.__getitem__`, catch the `KeyError`, and raise an `AttributeError` instead?